controllers/newsControllers.js

```diff
@@ -1,6 +1,15 @@
-const { fetchTopics } = require("../models/newsModels");
+const { fetchTopics, fetchArticle } = require("../models/newsModels");
 exports.getTopics = (req, res, next) => {
   return fetchTopics().then((topics) => {
     return res.status(200).send({ topics });
   });
 };
+
+exports.getArticle = (req, res, next) => {
```

Comment:
I think getArticleById is a more semantic name for this function. This is not an urgent fix, but just something to consider when naming your functions